### PR TITLE
Unify Thymeleaf Template Messages for Consistency

### PR DESCRIPTION
**Pull request by iishitahere**
This PR refactors the error page to ensure consistent message formatting by implementing Bootstrap alerts. It unifies the message display in the Thymeleaf template, improving readability and alignment with project standards.

### CODE REVIEW

File: src/main/resources/templates/error.html [hunk 1]
```diff
@@ -1,25 +1,32 @@
...
-        <br/><br/>
+<!-- Insert Header Fragment -->
+<div th:insert="~{fragments/header}"></div>
+
+<!-- Spacing Div -->
+<div class="space-div"></div>
+
+<!-- Standardized Message Format -->
+<div align="center" th:if="${data}">
+    <div th:data-testid="petListMessage"
```
Comment (josdem) [inline src/main/resources/templates/error.html:22]: `data-testid` id is normally used in E2E testing, therefore we should use a better name here, I suggest to use `th:data-testid="errorMessage"`

**Review by josdem — CHANGES_REQUESTED** [review]
Hi @iishitahere, this PR needs work, I noticed we have still messages with the old format:
- https://github.com/josdem/vetlog-spring-boot/blob/d9c17d21663f83e776f9ae56b73554d091bb769f/src/main/resources/templates/home/home.html#L17L19
-  https://github.com/josdem/vetlog-spring-boot/blob/d9c17d21663f83e776f9ae56b73554d091bb769f/src/main/resources/templates/petlog/create.html#L17L20

File: src/main/resources/templates/error.html [hunk 2]
```diff
@@ -1,25 +1,32 @@
 <!DOCTYPE html>
 <html xmlns:th="http://www.thymeleaf.org">
 <head>
-    <title>Vetlog Error page</title>
-    <head th:insert="~{fragments/include}"/>
+    <title>Vetlog Error Page</title>
+    <!-- Include Bootstrap CSS -->
+    <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" rel="stylesheet">
```
[the remark this answers is not in the document]
Comment (iishitahere) [reply src/main/resources/templates/error.html]: Hi @josdem,

Thank you for your feedback. I’ve worked on the changes you requested:

Reverted to the project's specific Bootstrap version.
Updated the error page according to the standardized message format.
Changed the data-testid to errorMessage, as suggested.
I believe the changes should now meet the project's requirements. Please let me know if any further adjustments are needed.

Looking forward to your feedback!

**Review by kaminuma — COMMENTED** [review]
Hi @iishitahere  @josdem
I have reviewed the code.
In my environment, when I bring the forked branch to my local environment, the styles appear broken. As a result, I was unable to verify the behavior of the centered alignment locally. However, I believe the content of the discussion above has been reflected in the code.

That said, the comment 
Revert to the project's specific Bootstrap version 

[NIT]
seems to indicate that the changes in this update were reverted. 
If this is meant to restore the original state, I feel that this comment might be unnecessary to avoid future confusion.

**Comment by josdem** [comment]
> Hi @iishitahere @josdem I have reviewed the code. In my environment, when I bring the forked branch to my local environment, the styles appear broken. As a result, I was unable to verify the behavior of the centered alignment locally. However, I believe the content of the discussion above has been reflected in the code.
> 
> [NIT] That said, the comment Revert to the project's specific Bootstrap version
> 
> seems to indicate that the changes in this update were reverted. If this is meant to restore the original state, I feel that this comment might be unnecessary to avoid future confusion.

Thanks for keeping a look at it, I appreciate it!

File: src/main/resources/templates/error.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (josdem) [reply src/main/resources/templates/error.html]: > Hi @josdem,
> 
> Thank you for your feedback. I’ve worked on the changes you requested:
> 
> Reverted to the project's specific Bootstrap version. Updated the error page according to the standardized message format. Changed the data-testid to errorMessage, as suggested. I believe the changes should now meet the project's requirements. Please let me know if any further adjustments are needed.
> 
> Looking forward to your feedback!

Thanks, this comment needs to be addressed too: https://github.com/josdem/vetlog-spring-boot/pull/443#pullrequestreview-2471038105

**Comment by iishitahere** [comment]
> Hi @iishitahere @josdem I have reviewed the code. In my environment, when I bring the forked branch to my local environment, the styles appear broken. As a result, I was unable to verify the behavior of the centered alignment locally. However, I believe the content of the discussion above has been reflected in the code.
> 
> [NIT] That said, the comment Revert to the project's specific Bootstrap version
> 
> seems to indicate that the changes in this update were reverted. If this is meant to restore the original state, I feel that this comment might be unnecessary to avoid future confusion.

Thank you for the review and feedback!

I’ll look into the styling issue and ensure the changes are verified properly.
I’ll update or remove the comment about reverting the Bootstrap version to avoid any future confusion.
Please let me know if there’s anything else you'd like me to address!

**Review by kaminuma — APPROVED** [review]
LGTM

**Comment by josdem** [comment]
> Hi @iishitahere, this PR needs work, I noticed we have still messages with the old format:
> 
> * https://github.com/josdem/vetlog-spring-boot/blob/d9c17d21663f83e776f9ae56b73554d091bb769f/src/main/resources/templates/home/home.html#L17L19
> * https://github.com/josdem/vetlog-spring-boot/blob/d9c17d21663f83e776f9ae56b73554d091bb769f/src/main/resources/templates/petlog/create.html#L17L20

Hi, @iishitahere as I mentioned this task needs work, I can see still old message formats, please take a look at those links.

**Comment by josdem** [comment]
Closing this PR in favor of this PR: https://github.com/josdem/vetlog-spring-boot/pull/448

**Pull request closed without merging**